### DISCUSSION TITLE
Offer Builder: request to add new NFTs above existing ones

### DISCUSSION THREAD

**Discussion by danieljperry** [Chia-Network/chia-blockchain-gui#1085] [Ideas]
On 1.6.1 rc2, start with an offer for an NFT. Here are a few requests from this state:
* After clicking the `+` button, a new text box appears below the current NFT. Request to add the text box above the NFT (or at the top of the list if there are multiple NFTs on offer). The reason for this request is because the new dialog will appear off-screen  on most computers, so users might not realize it exists
* If there is an empty text box in this section, disable the `+` button. Currently a user can continue clicking the `+` button to create multiple empty text boxes, each of which must be filled in

**Comment by paninaro** [comment]
@sethjenks for visibility

**Reply by danieljperry** [reply]
Another related request:

Currently (1.6.1 RC5) if an NFT text box is empty (which may occur due the the aforementioned suggestions), and the user clicks "Create Offer", this is the resulting error:

`Error
Invalid NFT ID`

Users could get confused by this message, especially if they have already entered a legitimate NFT ID in a different part of the offer. The error isn't specific as to which NFT ID is invalid, so they might not realize it refers to the empty box.

Suggestion is either
1. Specify where the error is, likely by highlighting the empty text box
or
3. Don't throw an error at all. Simply ignore the empty text box
